lib/jobs/OperationStatusPollerJob.js

```diff
@@ -41,63 +54,71 @@ class OperationStatusPollerJob extends BaseJob {
+          const currTime = moment();
+          // 'backup_restore_status_poller_timeout' config data might need to put in job data: operation specific
+          // operation can be other than backup/restore : thought just for future reference
+          if (currTime.diff(operationStartedAt) > config.backup.backup_restore_status_poller_timeout) {
+            return ScheduleManager.cancelSchedule(`${deploymentName}_${operationName}_${backupGuid}`, CONST.JOB.OPERATION_STATUS_POLLER)
               .then(() => {
-                logger.info(`Deployment ${deploymentName} ${operationName} for backup guid ${backupGuid} completed -`, operationStatusResponse);
-                if (operationStatusResponse.state === 'succeeded') {
-                  this.runSucceeded(operationStatusResponse, job, done);
-                } else {
-                  const msg = `Deployment ${deploymentName} ${operationName} for backup guid ${backupGuid} ${operationStatusResponse.state}`;
-                  logger.error(msg);
-                  const err = {
-                    statusCode: `ERR_DEPLOYMENT_${operationName.toUpperCase()}_${operationStatusResponse.state.toUpperCase()}`,
-                    statusMessage: msg
-                  };
-                  this.runFailed(err, operationStatusResponse, job, done);
-                }
+                const msg = `Deployment ${deploymentName} ${operationName} with backup guid ${backupGuid} exceeding timeout time 
+                      ${config.backup.backup_restore_status_poller_timeout / 1000 / 60} (mins). Stopping status check`;
+                logger.error(msg);
+                return operationStatusResponse;
               });
           } else {
-            logger.info(`Deployment ${deploymentName} ${operationName} for backup guid ${backupGuid} still in-progress - `, operationStatusResponse);
-            const currTime = moment();
-            // 'backup_restore_status_poller_timeout' config data might need to put in job data: operation specific
-            // operation can be other than backup/restore : thought just for future reference
-            if (currTime.diff(operationStartedAt) > config.backup.backup_restore_status_poller_timeout) {
-              ScheduleManager.cancelSchedule(`${deploymentName}_${operationName}_${backupGuid}`, CONST.JOB.OPERATION_STATUS_POLLER)
-                .then(() => {
-                  const msg = `Deployment ${deploymentName} ${operationName} with backup guid ${backupGuid} exceeding timeout time 
-                      ${config.backup.backup_restore_status_poller_timeout / 1000 / 60} (mins). Stopping status check`;
-                  logger.error(msg);
-                  const err = {
-                    statusCode: 'ERR_BACKUP_TIME_OUT',
-                    statusMessage: msg
-                  };
-                  return this.runFailed(err, operationStatusResponse, job, done);
+            return operationStatusResponse;
```

Comment:
here set `job_cancelled:false` or it can set by default.

Reply:
done